doc/getting_started/getting_started.rst

```diff
@@ -201,11 +201,11 @@ SDK, follow the steps below.
    .. code-block:: console
 
       # On Linux/macOS
-      export GCCARMEMB_TOOLCHAIN_PATH="~/gcc-arm-none-eabi-5_3-2016q1/"
-      export ZEPHYR_TOOLCHAIN_VARIANT=gccarmemb
+      export GNUARMEMB_TOOLCHAIN_PATH="~/gcc-arm-none-eabi-5_3-2016q1/"
```

Comment:
Shouldn't all the references to filenames with ``gcc`` in them be changed to ``gnu`` too?